src/oncall/api/v0/teams.py

```diff
@@ -86,7 +86,14 @@ def on_get(req, resp):
         query = '%s WHERE %s' % (query, where_query)
 
     cursor.execute(query, query_values)
-    data = [r[0] for r in cursor]
+    data = [None]
+    if 'get_id' in req.params:
+        if req.get_param('get_id') == '1':
+            data = [(r[0],r[5]) for r in cursor]
```

Comment:
Let's do `{'name': 'monitoring', 'id': 123}` for consistency (rather than tuple)